pkg/algorithm/hived_algorithm.go

```diff
@@ -370,14 +346,15 @@ func (h *HivedAlgorithm) GetVirtualClusterStatus(vcn api.VirtualClusterName) api
 	return h.apiClusterStatus.VirtualClusters[vcn].DeepCopy()
 }
```

Comment:
If not found such vc, need to
panic(internal.NewBadRequestError(fmt.Sprintf(VC %v not found)?